src/server/worker.js

```diff
@@ -64,7 +64,9 @@ export function run(worker) {
   }), graphQLHandler);
 
   // server-side rendering for emails
-  app.get('/email', emailSSR);
+  if (!PROD) {
```

Comment:
When we need a new email, it helps to see and work on a static template using Oy.